tests/bofire/strategies/doe/test_design.py

```diff
@@ -288,13 +285,13 @@ def test_find_local_max_ipopt_fixed_experiments():
         ],
     )
 
-    with pytest.warns(ValueError):
```

Comment:
@jduerholt here the sampler complains that there is nothing to sample. I think the sampler is correct - it can just sample [[1,0,0],[0,1,0],..]. How should this be handled? Should the sampler refuse to provide a sample if only a single allowed one exist?

Reply:
Fixed it in main.